pkg/registry/event/rest.go

```diff
@@ -45,7 +45,7 @@ func NewStorage(registry generic.Registry) *REST {
 func (rs *REST) Create(ctx api.Context, obj runtime.Object) (runtime.Object, error) {
 	event, ok := obj.(*api.Event)
 	if !ok {
-		return nil, fmt.Errorf("invalid object type")
+		return nil, errors.NewInternalError(fmt.Errorf("received object is not of type event: %v", obj))
```

Comment:
Hm, does %v display the type? I didn't think so-- %T or %#v would be more useful in that case.